### ISSUE TITLE
Cloning a VI creates parent virtual folder in wrong location

### ISSUE THREAD

**Issue by MichaelAivaliotis** [opengds/OpenGDS#11]
@MikaelHolmstrom 
I have a VI which is located in a nested virtual folder. For example:
class.lvclass\virtual_folder\virtual_sub-folder.
If I clone a VI located in that folder, then the target VI is created at:
class.lvclass\virtual_sub-folder.

Since the target folder doesn't exist, then now I have a new virtual folder created. So I must then move the VI to the proper location and delete the extra folder.

**Comment by MikaelHolmstrom** [comment]
So virtual on virtual folder is not handles correct. I'll work on this.
BTW I'm thinking of dropping support for 2012 and 2013, so these fixes will be from in 2014,2015,2016 Okay?

**Comment by MichaelAivaliotis** [comment]
I'm fine with using 2014 and up. I use 2014 still on some projects, but nothing below that.

**Comment by MichaelAivaliotis** [comment]
@MikaelHolmstrom, I am giving an update to this, since I'm not sure if you fixed it or changed the behavior. It might also be that the original issue (as posted) posted was not describing the problem correctly. It seems that when a VI cloning process is executed. You look at the folder name that the original VI is located in and then use that name to create a virtual folder with that same name. Is this correct?

If this is the case then this behavior is not desirable. I would prefer if the virtual folder hierarchy of the source VI was preserved instead of following the on-disk folder name.

This is probably why my original post was wrong. I think in my assessment, I happened to be using the same names on disk as in my virtual folder names. This is why the behavior was as it was.

What do you think?

![Screen Shot 2019-10-12 at 11 31 09 PM](https://user-images.githubusercontent.com/7982554/66711890-5078cb80-ed49-11e9-8450-d5ae86017a11.png)

**Comment by MikaelHolmstrom** [comment]
You are right, and I guess the proper behavior should be to place the VI under the same "Real" sub-folder (if one exists) and under the source's Virtual Folder structure.
That means the new clone ends up in the same location as it's source. 
Do we need an option to skip creating sub-folders and Virtual folders?

**Comment by MichaelAivaliotis** [comment]
I think if you implement it as you describe, that would make sense, and is what the user would expect. For me, the addition of a configuration option is not needed. However, if you think there are use-cases for an alternate behavior that can be switch on-off, then that is your call.

In the Clone VI Dialog you already show the new editable VI path. So a user can change this if needed. Perhaps show the editable virtual path as well? Both these fields should be pre-filled with the expected new locations. - Just an idea. But is not needed IMO.
